sphinxcontrib/datatemplates/__init__.py

```diff
@@ -7,4 +7,8 @@
 
 def setup(app):
     LOG.info('initializing sphinxcontrib.datatemplates')
-    app.add_directive('datatemplate', directive.DataTemplate)
+    app.add_directive('datatemplate-json', directive.DataTemplateJSON)
+    app.add_directive('datatemplate-yaml', directive.DataTemplateYAML)
+    app.add_directive('datatemplate-csv', directive.DataTemplateCSV)
+    app.add_directive('datatemplate-xml', directive.DataTemplateXML)
```

Comment:
We should think a bit about naming. Are there any similar packages we could pattern the names on? Should they be "format-data-template" instead of placing the template at the end?

Reply:
Actually a datatemplate domain with datatemplate:yaml would be best, I think. But I don't understand domains well enough yet for implementing.

Reply:
Excellent point, yes, let's figure out how to do that.

Reply:
Created #23 for this.